projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -510,4 +510,55 @@ public function add_gutenberg_menus( $wp_admin = false ) {
 	public function should_link_to_wp_admin() {
 		return get_user_option( 'jetpack_admin_menu_link_destination' );
 	}
+
+	/**
+	 * Returns the current slug from the URL.
+	 *
+	 * @param object $screen Screen object (undocumented).
+	 *
+	 * @return string
+	 */
+	public function get_current_slug( $screen ) {
+		$slug = "{$screen->base}.php";
+		if ( '' !== $screen->post_type ) {
+			$slug .= "?post_type={$screen->post_type}";
```

Comment:
What if we use the `add_query_arg` util?

```suggestion
			$slug = add_query_arg('post_type', $screen->post_type, $slug );
```

Reply:
839d449